packages/testcontainers/src/wait-strategies/utils/port-check-real.test.ts

```diff
@@ -0,0 +1,49 @@
+import path from "path";
+import { getContainerRuntimeClient } from "../../container-runtime";
+import { GenericContainer } from "../../generic-container/generic-container";
+import { checkContainerIsHealthy } from "../../utils/test-helper";
+import { Wait } from "../wait";
+import { InternalPortCheck } from "./port-check";
+
+describe("PortCheck", { timeout: 120_000 }, () => {
+  describe("InternalPortCheck", () => {
```

Comment:
I don't think an integration test is necessary for the `PortCheck` in addition to the existing unit tests. There are integration tests which cover this functionality here: https://github.com/testcontainers/testcontainers-node/blob/7e302a510eb24999cc2df3916412c590fcfb13ab/packages/testcontainers/src/wait-strategies/host-port-wait-strategy.test.ts#L5-L28